src/assert.js

```diff
@@ -305,6 +305,112 @@ class Assert {
 			message
 		} );
 	}
+
+	rejects( promise, expected, message ) {
+		let result = false;
+
+		const currentTest = ( this instanceof Assert && this.test ) || config.current;
+
+		// 'expected' is optional unless doing string comparison
+		if ( objectType( expected ) === "string" ) {
+			if ( message === undefined ) {
+				message = expected;
+				expected = undefined;
+			} else {
+				message = "rejects does not accept a string value for the expected argument.\n" +
```

Comment:
Can we put `assert.rejects() does not accept a string value...`? I think it is just a bit clearer/more helpful.